config/sync/field.storage.paragraph.field_p_m_heading.yml

```diff
@@ -0,0 +1,21 @@
+uuid: dc31d27e-96e7-43d7-aa41-29655b581c85
+langcode: en
+status: true
+dependencies:
+  module:
+    - paragraphs
+id: paragraph.field_p_m_heading
+field_name: field_p_m_heading
+entity_type: paragraph
+type: string
+settings:
+  max_length: 255
+  is_ascii: false
+  case_sensitive: false
+module: core
+locked: false
+cardinality: 1
+translatable: true
+indexes: {  }
+persist_with_no_fields: false
+custom_storage: false
```

Comment:
Style the heading in the front end so that it actually looks like a heading and also ensure it's not required.